lib/std/json.zig

```diff
@@ -896,16 +898,11 @@ pub const TokenStream = struct {
             }
         }
 
-        if (self.i > self.slice.len) {
-            try self.parser.feed(' ', &t1, &t2);
-            self.i += 1;
-
-            if (t1) |token| {
-                return token;
-            }
+        if(self.parser.complete){
+            return null;
+        } else {
+            return error.Incomplete;
```

Comment:
Can I propose instead of `Incomplete`:

`UnexpectedEndOfJson`

This way if the error code is bubbled up a few times with `try` it will still make sense.

Reply:
Yeah, I wanted to name it better, but forgot.